### PR TITLE
add docs for firebase and token_auth

### PR DESCRIPTION
**Pull request by axveez**
ISSUE [#5093](https://github.com/fortytools/skippr/issues/5093)

### CODE REVIEW

**Comment by axveez** [comment]
image

### Firebase Doc

**Comment by axveez** [comment]
image

### Token Auth Doc

File: fortytools.yaml [hunk 1]
```diff
@@ -106,7 +106,62 @@ paths:
...
+        Performs a login to get firebase credentials
+      security: []
+      requestBody:
+        description: the firebase credentials you want to post
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                email:
```
Comment (bigsolom) [inline fortytools.yaml:123]: Can you also please describe which parameters are required? I think all of them are https://swagger.io/docs/specification/describing-parameters/

File: fortytools.yaml [hunk 2]
```diff
@@ -106,7 +106,62 @@ paths:
           description: The app identified by the corresponding query parameter is not installed for any client of the user
         403:
           description: Invalid credentials, there is no User with this email / password combination.
-
+  /auth/firebase:
+    post:
+      tags:
+      - Auth
+      description: |
+        Performs a login to get firebase credentials
```
Comment (bigsolom) [inline fortytools.yaml]: Maybe we can align the description of this endpoint with the other one? as they are both doing the same thing; authentication; just with different input parameters.

File: fortytools.yaml [hunk 3]
```diff
@@ -106,7 +106,62 @@ paths:
...
+      security: []
+      requestBody:
+        description: the firebase credentials you want to post
```
Comment (bigsolom) [inline fortytools.yaml]: we're not posting here firebase credentials but rather login credentials (email, password)

File: fortytools.yaml [hunk 4]
```diff
@@ -106,7 +106,62 @@ paths:
...
+                  type: string
+      responses:
+        200:
+          description: Send Firebase credentials including firebase_id and firebase_token
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/FirebaseCredential'
+        401:
+          description: Invalid credentials, there is no User with this email / password combination.
```
Comment (bigsolom) [inline fortytools.yaml]: there's no email/password in this request

**Review by bigsolom — CHANGES_REQUESTED** [review]

File: fortytools.yaml (hunk 1 above)
Comment (axveez) [reply fortytools.yaml:123]: updated

**Review by bigsolom — APPROVED** [review]

**Pull request merged**